### ISSUE TITLE
ORA-24816 in database queue (jobs table)

### ISSUE THREAD

**Issue by fsblemos** [yajra/laravel-oci8#170]
I'm trying to execute the following code:

```
$object = \App\Test::first();

$data = compact('object');

Mail::queue('mail.test', $data, function ($message) use ($mail, $name) {
         $message->to($mail, $name);
         $message->subject('Test message');
}
```

It throws ORA-24816 exception. Researching about it:

https://community.oracle.com/thread/417560

> ORA-24816: Expanded non LONG bind data supplied after actual LONG or LOB column
Cause: A Bind value of length potentially > 4000 bytes follows binding for LOB or LONG.
**Action: Re-order the binds so that the LONG bind or LOB binds are all at the end of the bind list.**

It seems that this happens due to the size of `$data`, which is loaded in the CLOB "payload" column of the table "jobs". According to the action mentioned above, this column should be the last one on insert statement. 

I did this in Illuminate\Queue\DatabaseQueue, in buildDatabaseRecord function by putting the "payload" at the end of the array. It worked!

But I think the better way to solve this permanently should be the laravel-oci8 to put the CLOBs at the end by default.

**Comment by yajra** [comment]
@fsblemos thanks for taking time to look on this issue and sending a PR to laravel framework to fix this. Issue is the same on #164 and might also be related to this [issue](https://github.com/yajra/pdo-via-oci8/issues/24)

**Comment by yajra** [comment]
@fsblemos, PR was merged. Lets wait for it to tagged and will close the issue. Thanks!

**Comment by yajra** [comment]
Your PR was released on [v5.2.32](https://github.com/laravel/framework/commit/7c8d3d04e51cace016474aea8a6ab9a8682f4ab8). Thanks!
